app/services/hmpps_api/prison_api/movement_api.rb

```diff
@@ -40,7 +43,7 @@ def self.admissions_for(offender_nos)
                      .group_by { |x| x['offenderNo'] }.values
 
         movements = data.map { |d|
-          d.sort_by { |k| k['createDateTime'] }.map { |movement|
```

Comment:
Should we be sorting by `movementDate` if this can be `nil`?

Any reason why we need to change this sort at this point?

Reply:
turned out that movementDate had been broken for 2+ years, fixed on 19-Jan-2021